packages/mongo-menu/src/MongoMenu.story.tsx

```diff
@@ -3,16 +3,19 @@ import { storiesOf } from '@storybook/react';
 import { text, select } from '@storybook/addon-knobs';
 import MongoMenu, { Product } from '.';
 
-storiesOf('MongoMenu', module).add('Default', () => (
-  <MongoMenu
-    user={{
-      name: text('name', 'Alex Smith'),
-      email: text('email', 'alex.smith@mongodb.com'),
-    }}
-    activeProduct={select(
-      'activeProduct',
-      Object.values(Product) as Array<Size>,
-      Product.Atlas,
-    )}
-  />
-));
+const user = {
+  name: text('name', 'Alex Smith'),
+  email: text('email', 'alex.smith@mongodb.com'),
+};
+
+const activeProduct = select(
+  'activeProduct',
+  Object.values(Product) as Array<Product>,
+  Product.Atlas,
+);
+
+storiesOf('MongoMenu', module)
+  .add('Default', () => <MongoMenu user={user} activeProduct={activeProduct} />)
+  .add('Disabled MongoDB Account link', () => (
```

Comment:
We're trying to refrain from adding this sort of thing to the sidebar. If you remove this addition to the story, I think the PR is good to go 👍